torch_xla/csrc/tensor_util.cpp

```diff
@@ -130,16 +130,16 @@ struct Caster<xla::half> {
   }
 };
 template <>
-struct Caster<std::complex<float>> {
```

Comment:
Cannot replace, as this is also used to cast XLA data, which uses `std::complex`.

Reply:
Hi @dlibenzi, thanks for the hint! I am a bit confused by how this Caster work: what would happen if the casting is from `std::complex<double>` to `std::complex<float>`? Will it discard the imag part of the number?